README.md

```diff
@@ -65,4 +65,21 @@ For Snapshot and CDC modes, the following privileges are required:
 
 ## Destination
 
-(Planned)
+The mysql destination takes a `record.Record` and parses it into a valid SQL query. Each record is individually parsed and upserted, and write batching is planned to be implemented.
+
+### Upsert Behavior
+
+If the target table contains a column with a unique constraint (this includes PRIMARY KEY and UNIQUE indexes), records will be upserted; otherwise, they will be appended. Support for updating tables without unique constraints is tracked [here](https://github.com/conduitio-labs/conduit-connector-mysql/issues/66).
+
+### Multicollection mode
+
+(Planned to do). You can upvote [the following issue](https://github.com/conduitio-labs/conduit-connector-mysql/issues/13) to add more interest on getting this feature implemented sooner.
+
+### Configuration Options
+
+| name    | description                                                                                                                                                                           | required | default                                      |
+| ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | -------------------------------------------- |
+| `url`   | Connection string for the mysql database.                                                                                                                                          | true     |                                              |
```

Comment:
mysql -> MySQL. Also, it would be good to drop a link to the docs for the connection strings, just for convenience.

Reply:
fixed, added an example instead, should be more straightforward.

Reply:
An example is a great start for a user, but having a link means that the user can find out about all of the available options.